src/components/NcListItem/NcListItem.vue

```diff
@@ -386,7 +399,8 @@
 					<div class="list-item-content">
 						<div class="list-item-content__main">
 							<div class="list-item-content__name">
-								{{ name }}
+								<!-- @slot Slot for the first line of the component. prop 'name' is used as a fallback is no slots are provided -->
+								<slot name="title">{{ name }}</slot>
```

Comment:
> is `<template #name>` considered as OK for Vue?

Yes. Why?

Reply:
Just doesn't feel correctly somewhere deep inside in me: `<slot name="name"></slot>`

But I'm fine, if it's aligned with a `name` prop and maintains the same functionality (see `computedActionsAriaLabel`)

Reply:
Ah, I also thought that name was a bit off, as we already have a name prop, however, even in the original body of the PR, I don't have any clue on what to properly call it. Header would not be right, as then it would not fit the subname slot. I agree though that it should be name :)

Reply:
It definitely must be named `name` because is represents exactly the same content that `name` prop. So you have something called `name` in the `ListItem`, and you have two interfaces to set the name - the prop and the slot.